totalRP3/modules/register/characters/register_characteristics.lua

```diff
@@ -791,6 +791,16 @@ local function onMiscInfoDragStart(handle)
 	local ticker = C_Timer.NewTicker(MISC_INFO_DRAG_UPDATE_PERIOD, onMiscInfoDragUpdate);
 	ticker.handle = handle;
 	handle.miscInfoTicker = ticker;
+
+	-- Stick the icon of the item in question onto the cursor for some feedback.
+	-- Also throw in some sound cues a-la spellbook drag/drop.
+	local node = handle.node;
+	SetCursor(node.Icon.Icon:GetTexture());
+
+	-- For some reason there's no constant for the pickup sound effect
+	-- used by ability icons. Logically this'd be present as
+	-- IG_ABILITY_ICON_PICKUP.
+	PlaySound(837);

```

Comment:
I **really love** the idea of sound feedback!
But then you should use the `TRP3_API.ui.misc.playUISound(soundID)` function. The other dev who worked with me on Total RP 3 insisted that if we have UI sounds we have an option to disable them in the settings, so using that function will follow that setting :)